cumulusci/tasks/salesforce/PackageUpload.py

```diff
@@ -116,6 +111,35 @@ def _run_task(self):
                 )
             )
 
+    def _handle_package_upload_error(self):
+        self.logger.error("Package upload failed with the following errors")
+        error = {"message": ""}
+        apex_test_failures = False
+        for error in self.upload["Errors"]["errors"]:
+            self.logger.error("  {}".format(error["message"]))
+            if error["message"] == "ApexTestFailure":
+                apex_test_failures = True
+
+        if apex_test_failures:
+            self.logger.error("Failed Apex Tests:")
+            test_start_datetime = datetime.utcnow() - timedelta(
+                seconds=self.upload_time_seconds
+            )
```

Comment:
You're calculating the elapsed time using the start time, and then subtracting it from now to get the start time. Why not just store the start time and use it?